kea-vscode/src/views/common/remote-commit/remote-file-tree-node.ts

```diff
@@ -0,0 +1,44 @@
+import * as vscode from "vscode";
+import { IAccountKey } from "../../../account/account";
+import { IKeaContext } from "../../../core/context";
+import { createGitDecorationUri } from "../../../decorations/decoration-schemes";
+import { CommitFile, FileComment, RepoId } from "../../../types/kea";
+import { BaseFileTreeNode } from "../../common/base-file-tree-node";
+
+/**
+ * Tree item representing a remote file.
+ */
+export class RemoteFileTreeNode extends BaseFileTreeNode {
+  #resourceUri: vscode.Uri;
+
+  constructor(accountKey: IAccountKey, repoId: RepoId, file: CommitFile, comments: FileComment[], ctx?: IKeaContext) {
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    const fileName = file.filename.split("/").pop()!;
+    const fileUri = vscode.Uri.file(file.filename);
+
+    super(fileName, fileUri, comments, "file", ctx);
+
+    this.#resourceUri = createGitDecorationUri({
+      accountKey,
+      filePath: file.filename,
+      repoId,
+      fileStatus: file.status,
+    });
+  }
+
+  getTreeItem = (): vscode.TreeItem => {
+    const treeItem = this.createBaseTreeItem();
+
+    if (this.ctx) {
+      treeItem.command = this.ctx.commandManager.getCommand("kea.openCommitFileDiff", "Open File Diff", { resourceUri: this.#resourceUri });
+    } else {
+      treeItem.command = {
+        command: "kea.openCommitFileDiff",
+        title: "Open File Diff",
+        arguments: [{ resourceUri: this.#resourceUri }],
+      };
+    }
+
+    return treeItem;
+  };
```

Comment:
The left-hand side of the file matches the right-hand side. It's not fetching the file from the previous commit.

Reply:
This still does not work. Please check all the diffs in case it's not just this one.